timetracker/account/urls.py

```diff
@@ -22,4 +21,11 @@
         views.ProfileView.as_view(),
         name='profile',
     ),
+    path(
+        'signup/',
+        views.SignUpView.as_view(),
+        name='signup',
+        ),
```

Comment:
Stylistically, this last line should be indented to the level of `path`.

```suggestion
    ),
```